bute/theme.json

```
@@ -724,6 +724,7 @@
 				}
 			},
 			"core/navigation": {
+				"css": " .wp-block-navigation__responsive-container.is-menu-open{justify-content:center;}}",
```

```suggestion
				"css": " .wp-block-navigation__responsive-container.is-menu-open{justify-content:center;}",
```

I think there's an accidental extra bracket here.